src/test/java/com/aspectsecurity/contrast/contrastjenkins/ThresholdConditionStub.java

```diff
@@ -4,7 +4,7 @@
 public class ThresholdConditionStub extends ThresholdCondition {
 
     public ThresholdConditionStub() {
-        super(0, "test", "test");
+        super(0, "test", "test", "WebGoat");
```

Comment:
Is `WebGoat` supposed to be in here?

Reply:
The app name is required for the new constructor. Changed to "test" like the rest of the passed parameters.